test-files/cmd/glacial/ConflateCmdHighwayExactMatchInputsTest.sh

```diff
@@ -4,6 +4,6 @@ set -e
 mkdir -p test-output/cmd/ConflateCmdHighwayExactMatchInputsTest
 cp test-files/DcTigerRoads.osm test-files/DcTigerRoadsHighwayExactMatchInputs.osm
 
-hoot conflate -D match.creators="hoot::HighwayMatchCreator" -D merger.creators="hoot::HighwayMergerCreator" -D "writer.include.debug.tags=true" -D "writer.include.conflate.score.tags=true" -D "uuid.helper.repeatable=true" test-files/DcTigerRoads.osm test-files/DcTigerRoadsHighwayExactMatchInputs.osm test-output/cmd/ConflateCmdHighwayExactMatchInputsTest/output.osm
+hoot conflate -D hash.seed.zero=true -D match.creators="hoot::HighwayMatchCreator" -D merger.creators="hoot::HighwayMergerCreator" -D "writer.include.debug.tags=true" -D "writer.include.conflate.score.tags=true" -D "uuid.helper.repeatable=true" test-files/DcTigerRoads.osm test-files/DcTigerRoadsHighwayExactMatchInputs.osm test-output/cmd/ConflateCmdHighwayExactMatchInputsTest/output.osm
```

Comment:
I'm thinking that maybe we should extract this into a new file `conf/core/Testing.conf` or something like that (similar to `conf/core/NetworkAlgorithm.conf`) and then change this to be `-C Testing.conf` and put it into all of the other tests.  Then if we come across another setting that isn't default for regular conflation but it is for testing we can add it in there too.

Reply:
Good idea, done.

Reply:
That looks good.